scripts/package/package-debian.sh

```diff
@@ -115,10 +115,6 @@ parseargs(){
         echo "Provide the NuGet name of the targetted Shared Framework. Missing option '--framework-nuget-name'" && help
     fi
 
-    if [ -z "$PREVIOUS_VERSION_URL" ]; then
```

Comment:
Ok, can we remove the commented out section below as well then?  So it is completely out of the repo?